openapi.yaml

```diff
@@ -27258,6 +27668,130 @@ components:
           x-linode-cli-display: 6
           example: '2018-01-01T13:46:32'
           readOnly: true
+    PlacementGroup:
+      type: object
+      properties:
+        id:
+          type: integer
+          nullable: false
+          description: >
+            The placement group's ID. You need to provide it for all operations impacting it.
+          example: 528
+        label:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          description: |
+            The unique name set for the placement group. A label has these constraints:
+
+            * It needs to begin and end with an alphanumeric character.
+            * It can only consist of alphanumeric characters, hyphens (`-`), underscores (`_`) or periods (`.`).
+          example: PG_Miami_failover
+          minLength: 1
+        region:
+          type: string
+          x-linode-cli-display: 1
+          x-linode-filterable: true
+          readOnly: true
+          description: >
+            The [region](/docs/api/regions/#regions-list) where the placement group was deployed.
+          example: us-mia
+        affinity_type:
+          type: string
+          x-linode-filterable: true
+          readOnly: true
+          description: |
+            How compute instances are distributed in your placement group. `anti-affinity:local` places compute instances in separate fault domains, but still in the same region.
+
+            **Note**: With the beta release, only `anti_affinity:local` is available for `affinity_type`.
+          example: anti-affinity:local
+        is_compliant:
+          type: boolean
+          nullable: false
+          description: >
+            The availability of the placement group. `true` indicates the group is available and all the compute instances in your group meet your `affinity_type` and `is_strict` enforcement setting. If `false`, the group is unavailable. One or more compute instances violate your `affinity_type` and `is_strict` settings. You need to wait for our assistance to physically move compute instances to make the group compliant.
+          example: true
+        members:
```

Comment:
We might be able to use the `x-linode-cli-subtables` extension here to display this info in a separate table, but this should be fine for now 👍

Reply:
@lgarber-akamai ... if you tell me what I need to add, I can throw it in there.

Reply:
@Vernholio I'll take a look into this shortly since subtables can be a bit finicky to implement

Reply:
If this is going to be a lot of work/difficult to implement, we can address it in a future release.

Reply:
Sounds good!